module/VuFind/src/VuFind/Bootstrapper.php

```diff
@@ -241,12 +241,20 @@ protected function initUserLanguage(): void
      */
     protected function initTheme(): void
     {
-        // Attach remaining theme configuration to the dispatch event at high
-        // priority (TODO: use priority constant once defined by framework):
+        // Attach remaining theme configuration to the dispatch event at high priority:
         $config = $this->config->Site;
         $callback = function ($event) use ($config) {
             $theme = new \VuFindTheme\Initializer($config, $event);
-            $theme->init();
+            try {
+                $theme->init();
+            } catch (\Exception $e) {
+                // Try to display an error page if the theme fails to initialize:
+                $config = $this->container->get('config');
```

Comment:
Re-use of `$config` (first defined on line 246) is a bit confusing; maybe `$moduleConfig` or something?

Reply:
Good point. 3fd2558111c9b6d71213fbb5acacb548edd20ffa renames both of the variables for improved specificity.